### PR TITLE
Update Print Preview Layout

### LINKED ISSUE #2182: Update Print Preview Layout
Update the print preview as described in design task #2117 

### This task is done when...
 - [ ] Moved the export/submit instructions from the bottom of this page to the top of the page
 - [ ] Added a < Back to eAPD link at the top so folks can get out of this screen
 - [ ] Added a ^ Return to the top of the page link at the bottom of this screen so folks can jump back to the top when they review the page in full
 - [ ] Increase the font size on the text instructions in the blue box to 18px (headers are fine at 21px)
 - [ ] Added a submit button
 - [ ] Icons changed to icons that are in the design system for the export button and the submit button  (https://design.cms.gov/style/icons/)

Note from designers: we have the print preview hiding the instructions that are being moved to the top. Let's make sure that stays hidden on the print PDF.

### PR DESCRIPTION
**Pull request by pphillips-fearless**
Update the print preview layout

### This pull request changes...

- Moved the export/submit instructions from the bottom of this page to the top of the page
- Added a < Back to eAPD link at the top so folks can get out of this screen
- Added a ^ Return to the top of the page link at the bottom of this screen so folks can jump back to the top when they review the page in full
- Increase the font size on the text instructions in the blue box to 18px (headers are fine at 21px)
- Added a submit button
- Icons changed to icons that are in the design system for the export button and the submit button (https://design.cms.gov/style/icons/)
- Print instructions are still hidden on the print PDF
- closes #2182

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
~~- [ ] The change has been documented~~
~~- [ ] Associated OpenAPI documentation has been updated~~
- [x] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by pphillips-fearless** [comment]
@CSwartzHMA adding a comment because you may have a vested interest in how this looks.

**Comment by beparticular** [comment]
Everything looks good on this except what's happening on the back to APD link. It's going back to the beginning, which is fine, but it's still highlighting the Export and Submit label on the nav, much like Caryn was seeing on the bug on #2223. 

If folks are comfortable, we could release this, but that bug is becoming more important to look at as well. Thoughts @pphillips-fearless and/or @jeromeleecms ?

**Comment by pphillips-fearless** [comment]
@beparticular I saw the issue you referenced while working on this, but knew we had #2223 on the books to fix it.  Agreed, that issue should be prioritized.

**Comment by jeromeleecms** [comment]
@beparticular Agreed - it needs to be fixed. Also, I don't see this as a sev 1 or sev 2 bug - the user can still complete the experience + we will be resolve this issue + it's being tracked.

**Comment by mgwalker** [comment]
@beparticular One question (I'm still in the code review, which shouldn't take much longer), but I noticed the blue help text boxes got bigger in both the builder mode and the print mode. Was it supposed to be changed for both, or only for print?

File: web/src/components/Icons.js [hunk 1]
```diff
@@ -33,7 +33,10 @@ const CheckCircle = ({ ...props }) => (
   <FontAwesomeIcon icon={faCheckCircle} {...props} />
 );
 const File = ({ ...props }) => <FontAwesomeIcon icon={faFileAlt} {...props} />;
-const FileDownload = () => <FontAwesomeIcon icon={faFileDownload} />;
+const Envelope = () => <FontAwesomeIcon icon={faEnvelope} />;
+const FileDownload = () => (
+  <img src="../static/img/pdf_doc.svg" alt="PDF" width="16em" />
+);
```
[the remark this answers is not in the document]
Comment (jeromeleecms) [reply web/src/components/Icons.js:39]: Yes, the plan is still to allow uploads of contract/other documents in the future (it's on the roadmap at the end). Based on user feedback, we will still need to maintain the export function (mainly for internal state review) even after we implement the electronic/online submission.

**Comment by mgwalker** [comment]
Talked to Jenn about the instruction box comment above, looks like we're good to go on that! 👍

File: web/src/components/Icons.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (pphillips-fearless) [reply web/src/components/Icons.js:39]: I like the name change

**Pull request merged**